core/scale/fixedwidth.cpp

```diff
@@ -9,71 +9,70 @@
 
 #include "scale/util.hpp"
 
```

Comment:
using common::ByteStream; could help improve formatting and readability, the same for common::Buffer;

Reply:
IMO formatting is fine